datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -188,6 +188,9 @@
     'terawatt',
     'heap',
     'volume',
+    'milliwatt',
+    'microwatt',
+    'nanowatt'
```

Comment:
```suggestion
    'nanowatt',
```

^ style is failing on this